### ISSUE TITLE
What does 's' mean?

### ISSUE THREAD

**Issue by wrpha-group** [sacmehta/ESPNetv2#10]
What does the network scale parameter 's' set during training mean?
CUDA_VISIBLE_DEVICES = 0, 1, 2, 3 python main.py - batch_size 10 - s 1.0

On lines 34-37 of SegmentationModel.py
`
if s <= 0.5:
`
` 
  p = 0.1
`
`
else:
`
` 
  p = 0.2
`

and line 46
`
self.project_l1 = nn.Sequential (nn.Dropout2d (p = p), C (self.net.level1.act.num_parameters + classes, classes, 1, 1))
`

I interpreted 's' as a parameter of dropout. However, looking at lines 34-37, it seems that there is no difference as s = 1.0 and s = 1.5 both p = 0.2.

What does 's' mean?

**Comment by sacmehta** [comment]
s is the scaling parameter which allows to grow or shrink the network width.

**Comment by wrpha-group** [comment]
Do you mean the number of repetitions of EESP unit?

**Comment by sacmehta** [comment]
Sorry I meant network width and not network depth.

**Comment by wrpha-group** [comment]
I am sorry to ask you many questions.
Does width mean the number of branches in EESPunit?

**Comment by sacmehta** [comment]
Width means number of output channels. Let us assume that we have a convolution layer that takes RGB image as an input and produces 10 channels as output. Assume that this configuration is at scale 1.0. When I use a scale of 0.5, then output channels will be scaled by 0.5 too i.e RGB image will be transformed from 3 channels to 5 channels now.

**Comment by wrpha-group** [comment]
I was misunderstanding.
Thank you for your polite answer!
